db/migrate/20181128175719_add_not_null_to_external_service_id_in_exports.rb

```diff
@@ -0,0 +1,5 @@
+class AddNotNullToExternalServiceIdInExports < ActiveRecord::Migration[5.2]
```

Comment:
The following 2 migrations enforce that we must have an external_system_id for exports and exported_files now that we have corrected the data